handbook/README.md

```diff
@@ -1,11 +1,24 @@
 ---
 title: Introduction
-...
+---
+
+The SystemLink Operations Handbook contains documentation, examples, and
+example config files for IT professionals to use when managing an
+[NI SystemLink Server](https://www.ni.com/systemlink) installation beyond the
+basic configuration installed with SystemLink. For information about how to
+install and use a SystemLink server, see the
+[latest product manual on ni.com](https://www.ni.com/r/systemlinkmanual).
+Source and contributions for the handbook and example files can be found in the
+[systemlink-operations-handbook GitHub repository](https://github.com/ni/systemlink-operations-handbook).
 
-# NI SystemLink Operations Handbook
+While the handbook is not specific to any version of SystemLink Server, the
+product continues to evolve over time. Some chapters or examples may
+specify a version or range of versions of SystemLink that they apply to.
 
-Placeholder
+## Table of Contents
 
-## Chapters
```

Comment:
I wonder if this section is necessary. Can we assume most people understand this?

Reply:
Are you referring to the heading, the text describing the section, or the entire section?

Reply:
The text is probably not needed.

Reply:
My b -- the text describing the section : )